packages/map-template/src/components/Map/Map.jsx

```diff
@@ -206,14 +206,14 @@ function Map({ onLocationClick, onVenueChangedOnMap }) {
         if (positionControl.nodeName === 'MI-MY-POSITION') {
             // The Web Component needs to set up the listener with addEventListener
             positionControl.addEventListener('position_received', positionInfo => {
-                if (positionInfo.accurate === true) {
-                    setUserPosition(positionInfo.position);
+                if (positionInfo.detail.accurate === true) {
+                    setUserPosition(positionInfo.detail.position);
                 }
             });
         } else {
             positionControl.on('position_received', positionInfo => {
-                if (positionInfo.accurate === true) {
-                    setUserPosition(positionInfo.position);
+                if (positionInfo.detail.accurate === true) {
+                    setUserPosition(positionInfo.detail.position);
```

Comment:
we don't need the `.detail` here - the `else` part of the main condition is checking if the my-position button is put in as an mi-component or if it is using the SDK's standard my-position button. In the latter case, the `positionInfo.accurate` and  `positionInfo.position` should work.

Reply:
Oh, alright. Removed the unnecessary part